### ISSUE TITLE
Version mismatch in ppa

### ISSUE THREAD

**Issue by baelter** [rsyslog/rsyslog-pkg-ubuntu#59]
Trying to install on ubuntu 14.04

This is what I'm doing:
```
sudo add-apt-repository ppa:adiscon/v8-stable
sudo apt-get update
sudo apt-get install rsyslog
```
build librdkafka 9.5 from source
```
sudo apt-get install rsyslog-kafka
```

But this happens:

```
 Reading package lists... 
 Building dependency tree... 
 Reading state information... 
 Some packages could not be installed. This may mean that you have 
 requested an impossible situation or if you are using the unstable 
 distribution that some required packages have not yet been created 
 or been moved out of Incoming. 
 The following information may help to resolve the situation: 
 The following packages have unmet dependencies: 
  rsyslog-kafka : Depends: rsyslog (= 8.26.0-0adiscon2trusty1) but 8.27.0-0adiscon2trusty1 is to be installed 
```

Additional info:

```
sudo apt-cache madison rsyslog-kafka
rsyslog-kafka | 8.26.0-0adiscon2trusty1 | http://ppa.launchpad.net/adiscon/v8-stable/ubuntu/ trusty/main amd64 Packages
sudo apt-cache madison rsyslog
   rsyslog | 8.27.0-0adiscon2trusty1 | http://ppa.launchpad.net/adiscon/v8-stable/ubuntu/ trusty/main amd64 Packages
   rsyslog | 7.4.4-1ubuntu2.6 | http://archive.ubuntu.com/ubuntu/ trusty-updates/main amd64 Packages
   rsyslog | 7.4.4-1ubuntu2.3 | http://archive.ubuntu.com/ubuntu/ trusty-security/main amd64 Packages
   rsyslog | 7.4.4-1ubuntu2 | http://archive.ubuntu.com/ubuntu/ trusty/main amd64 Packages
```

**Comment by baelter** [comment]
devel PPA is working. But I'd rather use "stable"

**Comment by baelter** [comment]
Was working earlier today. Probably related to 8.27.0 re-pack 61f4c3c5de48ae39ae13abbdc3aa1b072aba2ef1

**Comment by baelter** [comment]
Solved by 4fec6c3
